go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/valyala/fasthttp
+module github.com/lovromazgon/fasthttp
```

Comment:
We obviously can't merge this pull request with this change.

Reply:
Sorry, removed it!